pkg/tunnel/healthz.go

```diff
@@ -0,0 +1,62 @@
+package tunnel
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/paralus/paralus/pkg/grpc"
+	rpcv3 "github.com/paralus/paralus/proto/rpc/scheduler"
+	commonv3 "github.com/paralus/paralus/proto/types/commonpb/v3"
+	infrav3 "github.com/paralus/paralus/proto/types/infrapb/v3"
+	"github.com/paralus/relay/pkg/utils"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func syncClusterHeath(sni string, status commonv3.ParalusConditionStatus, reason string) {
+	id, err := getClusterID(sni)
+	if err != nil {
+		_log.Error("unable to get clusterID", "error", err)
+	}
+
+	u, err := url.Parse(utils.PeerServiceURI)
+	if err != nil {
+		_log.Error("unable to parse peer service url", "error", err)
+	}
+	//Load certificates
+	tlsConfig, err := ClientTLSConfigFromBytes(utils.PeerCertificate, utils.PeerPrivateKey, utils.PeerCACertificate, u.Host)
+	if err != nil {
+		_log.Error("unable to build tls config for peer service", "error", err)
+	}
```

Comment:
Same as above

Reply:
same thought for all error cases in cluster health update flow.